app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,7 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+    <%# <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
   </head>
```

Comment:
- [ ] Please make sure that the app displays a random greeting every time. At the moment, it only displays one greeting.
- [ ] I'd recommend adding greetings in the `seed` file to make it easier to populate the database. Kindly make sure that the database is populated with greetings and that a random greeting is fetched each time.


![image](https://github.com/Habtamu210/hello-rails-react/assets/62290545/5cd282a7-b213-47b5-a981-fdd178804f39)